src/components/pager/__internal__/pager-navigation.component.tsx

```diff
@@ -38,32 +81,37 @@ const PagerNavigation = ({
     onPagination,
   };
 
-  const handlePageInputChange = (ev) => {
+  const handlePageInputChange = (
+    ev:
+      | React.KeyboardEvent<HTMLInputElement>
+      | React.FocusEvent<HTMLInputElement>
+  ) => {
+    const { value } = ev.target as HTMLInputElement;
     if (pageCount === 0) {
       setCurrentPage(0);
       onPagination(0, pageSize, "input");
       return "0";
     }
 
-    if (Number(ev.target.value) <= 0 || Number.isNaN(Number(ev.target.value))) {
+    if (Number(value) <= 0 || Number.isNaN(Number(value))) {
```

Comment:
I think we would be better to be consistent here and either use `Number` or `+` rather than both